core/config.class.inc.php

```diff
@@ -1595,6 +1595,14 @@ class Config
 			'source_of_value'     => '',
 			'show_in_conf_sample' => false,
 		],
+		'security.single-profile-completion' => [
```

Comment:
```suggestion
		'security.single_profile_completion' => [
```

We use underscores instead of dashes.

Reply:
ok i will rename it